scripts/run_olserver.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This script git pulls master for openlibrary, infogami, and olsystem
+# and then starts the Docker service specified by $SERVICE
+
+cd /opt/olsystem
+sudo git pull origin master
+cd /opt/openlibrary
+sudo git pull origin master
+cd /opt/openlibrary/vendor/infogami
+sudo git pull origin master
+cd /opt/openlibrary
+
+export SERVICE=${SERVICE:-"web"}  # options: web, covers, infobase, home
```

Comment:
Don't need to export this, since we're not using it in any subshells.

```suggestion
SERVICE=${SERVICE:-"web"}  # options: web, covers, infobase, home
```